### ISSUE TITLE
Feature request: Create garment support from mesh

### ISSUE THREAD

**Issue by manavortex** [WolvenKit/Cyberpunk-Blender-add-on#106]
Current process of creating garment support: 
1. prepare mesh to create garment support _from_ (e.g. body, merge and shrink)
2. for each individual mesh,
- create shrinkwrap modifier
- target mesh from step 1
- apply as shapekey
- rename shapekey to GarmentSupport

I'm sure this can be automated? We'd just need a target picker.

Below are the settings I've had the best results with, we could use these as defaults

![image](https://github.com/WolvenKit/Cyberpunk-Blender-add-on/assets/965785/2d1ae3e0-0e78-4e52-942a-b12c8359957b)

In a second iteration, the plugin could offer base bodies and arms via drop-down, but that would require more resources. Please do not implement this in the first iteration, Presto <3 :D

**Comment by Simarilius-uk** [comment]
No. Garment support is the root of all evil.

**Comment by manavortex** [comment]
No, Auska defeated it, I've been using it for weeks without issues!
